trade_remedies_public/templates/registration/accessibility_statement.html

```diff
@@ -4,72 +4,131 @@
 
 {% block row_content %}
     <div class="terms-and-conditions-page">
-        <h1 class="heading-large"><span class="heading-secondary">Trade Remedies Digital Service</span>
-            Accessibility statement for the {% organisation_initialism %} service
-        </h1>
         <div class="column-two-thirds left-column">
-
-            <h2 class="heading-medium">Using this website</h2>
-            <p>This website is run by the {% organisation_name %} ({% organisation_initialism %}), part of Department
-                for International Trade (DIT). We want as many people as possible to use this website, so you should be
-                able to:</p>
+            <h2 class="heading-medium">Accessibility statement</h2>
+            <p>This statement applies to content published on <a href="www.traderemediesservice.gov.uk">www.traderemediesservice.gov.uk</a>.</p>
```

Comment:
This is a dead link.

```suggestion
            <p>This statement applies to content published on <a href="https://www.trade-remedies.service.gov.uk">www.trade-remedies.service.gov.uk</a>.</p>
```

Reply:
Done.